Modules/Core/Common/include/itkMacro.h

```diff
@@ -120,9 +120,18 @@ namespace itk
 #  define ITK_GCC_PRAGMA_DIAG_PUSH() ITK_GCC_PRAGMA_DIAG(push)
 #  define ITK_GCC_PRAGMA_DIAG_POP() ITK_GCC_PRAGMA_DIAG(pop)
 #else
-#  define ITK_GCC_PRAGMA_DIAG(x)
-#  define ITK_GCC_PRAGMA_DIAG_PUSH()
-#  define ITK_GCC_PRAGMA_DIAG_POP()
+#  define ITK_GCC_PRAGMA_DIAG(x)                                                                                       \
```

Comment:
It looks like "\" are missing here to me. If it is style compliant, putting it on one line may be clearer.

Reply:
I don't understand about the "".  Do you mean the end of line continuations "\\"?  They are there, but off the standard github display.  You need to scrol to the right.

I did write these as one liners, but clang-format expanded it to 3 lines.  Perhaps this is a good use of a comment to ignore clang-format.

Reply:
Sorry for the missing char! 😞 

I see the "\" now! That is very annoying that it is off the end of the GitHub display.